packages/p2p/src/components/composite-calendar/composite-calendar-mobile.jsx

```diff
@@ -0,0 +1,253 @@
+                                        key={duration.value}
+                                        value={duration.value}
+                                        label={duration.label}
+                                        selected_value={selected_date_range.value}
+                                        onChange={onDateRangeChange}
+                                    />
+                                ))}
+                        </div>
+                        <div className='composite-calendar-modal__custom'>
+                            <div className='composite-calendar-modal__custom__wrapper'>
+                                <RadioButton
+                                    id={'composite-calendar-modal__custom-radio'}
+                                    className='composite-calendar-modal__custom-radio'
+                                    value={CUSTOM_KEY}
+                                    label={localize('Custom')}
+                                    selected_value={selected_date_range.value}
+                                    onChange={onDateRangeChange}
+                                />
+                            </div>
+                            <div className='composite-calendar-modal__custom-date-range'>
+                                <DatePicker
+                                    className='composite-calendar-modal__custom-date-range-start-date'
+                                    is_nativepicker={true}
+                                    placeholder={localize('Start date')}
+                                    value={from}
+                                    max_date={max_date}
+                                    onChange={e => selectDate(e, 'from')}
+                                />
+                                <DatePicker
+                                    className='composite-calendar-modal__custom-date-range-end-date'
+                                    is_nativepicker={true}
+                                    placeholder={localize('End date')}
+                                    value={to}
+                                    max_date={today}
+                                    min_date={min_date}
+                                    onChange={e => selectDate(e, 'to')}
+                                />
+                            </div>
+                        </div>
+                        <Button
+                            className='composite-calendar-modal__actions-today'
+                            text={localize('Back to today')}
+                            onClick={selectToday}
+                            has_effect
+                            tertiary
+                            large
+                        />
+                    </div>
+                </MobileDialog>
+            </React.Fragment>
+        );
+    }
+);
+
+CompositeCalendarMobile.displayName = 'CompositeCalendarMobile';
```

Comment:
do we need to have this since the component already has the same name? this is similar throughout other components. If the component was named something different such as MyComponent, then this would be fine to use since you would do something like this

`MyComponent.displayName = 'CompositeCalendarMobile'`